Homework2.Tests/ArrayExtensionsTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Linq;
+using Xunit;
+
+namespace Homework2.Tests
+{
+    public class ArrayExtensionsTests
+    {
+        [Theory]
+        [InlineData(999, 4)]
+        [InlineData(999, 0)]
+        [InlineData(999, 10)]
+        public void AddAt_adds_at_idx(int item, int idx)
+        {
+            var testarray = new int[] { 11, 22, 33, 44, 55, 66, 77, 88, 99, 111 };
```

Comment:
Overkill test data. Having 3 elements would be enough.

Reply:
Made it smaller and used a ctor to init it.